compiler-plugin/src/main/kotlin/com/google/devtools/ksp/symbol/impl/kotlin/KSTypeArgumentLiteImpl.kt

```diff
@@ -39,5 +47,10 @@ class KSTypeArgumentLiteImpl private constructor(override val type: KSTypeRefere
 
     override val location: Location = NonExistLocation
 
+    override val parent: KSNode? by lazy {
+        (type as? KSTypeReferenceImpl)?.ktTypeReference
+            ?.findParentOfType<KtFunctionType>()?.let { KSCallableReferenceImpl.getCached(it) }
```

Comment:
why `KtFunctionType`?

Reply:
it is created from `KSCallableReferenceImpl`. Other use case is creating one from resolver therefore no `parent`